css/style.css

```diff
@@ -0,0 +1,244 @@
+:root {
+    --main-color: #2874f0;
+}
+
+html,
+body {
+    padding: 0%;
+    margin: 0%;
+}
+
+.bg-system {
+    background-color: var(--main-color) !important;
+}
+
+.navbar {
+    background-color: var(--main-color);
+    font-size: 1rem;
+    padding: 0% !important;
+}
+
+.navbar-container {
+    display: flex;
+    justify-content: center;
+    padding: 10px;
+    margin: 0 auto;
+}
+
+.navbar .bottom-part {
+    font-size: 11px;
+}
+
+.explore-link {
+    font-style: italic;
+    color: white;
+    text-decoration: none;
+}
+
+.explore-link:hover {
+    text-decoration: underline;
+}
+
+.yellow-text {
+    color: #ffe500;
+}
+
+.search-btn,
+.login-btn {
+    border: none;
+    padding: 10px;
+    background-color: white;
+    margin-right: 2vw;
+    max-height: 50px;
+}
+
+.search-btn {
+    margin-top: 5px;
+}
+
+.search-bar {
+    outline: none;
+    width: 25vw;
+    padding: 5px 15px;
+    border: none;
+    margin-left: 1vw;
+    max-height: 50px;
+    margin-top: 5px;
+}
+
+.search-icon {
+    font-weight: bolder;
+    color: var(--main-color);
+    padding: 0 10px;
+}
+
+.nav-link {
+    color: white !important;
+}
+
+.nav-item {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin: 0 1.2vw;
+}
+
+@media only screen and (max-width: 991px) {
+    .nav-item {
+        width: 100%;
+        display: initial;
+        text-align: center;
+        margin-bottom: 10px;
+    }
+
+    .dropdown-item {
+        text-align: center;
+    }
+
+    .login-btn {
+        width: 100% !important;
+    }
+}
+
+.login-btn {
+    width: 10vw;
+}
+
+.navbar .right-part h6 {
+    color: white;
+    padding-top: 10px;
+    margin-left: 60px;
+    cursor: pointer;
+}
+
+.category-section {
+    display: flex;
+    padding: 15px 0;
+    overflow: scroll;
+    justify-content: center;
+}
+
+@media screen and (max-width: 1240px) {
+    .category-section {
+        justify-content: start;
+    }
+}
+
+.category-section::-webkit-scrollbar {
+    display: none;
+}
+
+.category-section .item {
+    margin: 0 30px;
+    text-align: center;
+    cursor: pointer;
+    font-size: 1rem;
+}
+
+.category-section p {
+    margin-top: 0.1vw;
+    overflow: hidden;
+    white-space: nowrap;
+}
+
+.category-section .item:hover {
+    color: rgb(255, 0, 98);
+}
+
+.slider-btn {
+    margin-top: 4vw;
+    background-color: rgb(248, 249, 250) !important;
+    width: 3vw !important;
```

Comment:
why you use !important in each line for this class try to remove that

Reply:
Actually, in html I have used bootstrap but I also want to customize it so that's the reason I write !important on every property of CSS which I want to customize.